src/main/java/com/org/jenkins/custom/jenkins/distribution/service/generators/WarGenerator.java

```diff
@@ -16,7 +16,7 @@
     private static final String TEMP_PREFIX = "CDS";
 
 
-    public static File generateWAR(final String versionName, final String configuration) throws IOException, InterruptedException {
+    public File generateWAR(final String versionName, final String configuration) throws IOException, InterruptedException {
```

Comment:
Just question: did you remove static to make it easier to test?

Reply:
yes